### PR TITLE
Add link of translation

### PR DESCRIPTION
**Pull request by Jongjineee**
I added another translation link to the translated README.md

### CODE REVIEW

File: docs/ko-KR/TIPS.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<table>
+    <tr>
+        <!-- Do not translate this table -->
+        <td> Read these guidelines in </td>
+        <td><a href="/TIPS.md">English</a></td>
+        <td><a href="/docs/pt-BR/Dicas.md">Português</a></td>
+        <td><a href="/docs/es-ES/TIPS.md">Español</a></td>
+        <td><a href="/docs/ko-KR/TIPS.md">한국어</a></td>
```
Comment (Berkmann18) [inline docs/ko-KR/TIPS.md]: Could you please add the link to the French translation so at least there are fewer disconnections between the translation's navigation.

File: docs/ko-KR/other/CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+<table>
+    <tr>
+        <!-- Do not translate this table -->
+        <td> Read these guidelines in </td>
+        <td><a href="/other/CODE_OF_CONDUCT.md">English</a></td>
+        <td><a href="/docs/pt-BR/other/CODE_OF_CONDUCT.md">Português</a></td>
+        <td><a href="/docs/es-ES/other/CODE_OF_CONDUCT.md">Español</a></td>
+        <td><a href="/docs/ko-KR/other/CODE_OF_CONDUCT.md">한국어</a></td>
```
Comment (Berkmann18) [inline docs/ko-KR/other/CODE_OF_CONDUCT.md]: Same here (see my comment above).

File: docs/ko-KR/other/IMPLEMENTATIONS.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<table>
+    <tr>
+        <!-- Do not translate this table -->
+        <td> Read these guidelines in </td>
+        <td><a href="/other/IMPLEMENTATIONS.md">English</a></td>
+        <td><a href="/docs/pt-BR/other/IMPLEMENTATIONS.md">Português</a></td>
+        <td><a href="/docs/es-ES/other/IMPLEMENTATIONS.md">Español</a></td>
+        <td><a href="/docs/ko-KR/other/IMPLEMENTATIONS.md">한국어</a></td>
```
Comment (Berkmann18) [inline docs/ko-KR/other/IMPLEMENTATIONS.md]: Same here.

**Comment by MatheusRV** [comment]
why close?

**Pull request closed without merging**